src/tests/quantuminspire/qiskit/test_backend_qx.py

```diff
@@ -258,13 +259,22 @@ def test_get_experiment_results_multiple_single_shots(self):
         self.assertEqual(one_shot_results['0x2'], 4003)
         self.assertEqual(one_shot_results['0x3'], 984)
 
-    def test_validate_negative_shot_count(self):
+    def test_validate_shot_count_zero(self):
```

Comment:
It was never there, the name was misleading because the test was done with 0.
In the validation function itself the test is:
`if number_of_shots < 1:`
so 0 has the same result as any negative number